src/python/WMCore/Wrappers/JsonWrapper/JSONThunker.py

```diff
@@ -145,7 +133,7 @@ def handleDictThunk(self, toThunk):
         toThunk = self.checkRecursion(toThunk)
         special = False
         tmpdict = {}
-        for k, v in toThunk.iteritems():
+        for k, v in iteritems(toThunk):
```

Comment:
Dario, was there any reason for the iteritems choice (instead of viewitems)?

Reply:
You are right!
I fear that when editing this I had the impression that it was not safe to loop over a dynamic list of `toThunk`'s items, but now that I have another look at it `toThunk` is modified outside of the loop over its items and it is safe to use `viewitems`. I am sorry, I do not remember why I have been so conservative. Thanks for the thorough review!